pyscript.core/README.md

```diff
@@ -47,3 +47,46 @@ npm run server
 ### Dev Build
 
 Beside spinning the _localhost_ server via `npm run server`, the `npm run dev` will watch changes in the `./esm` folder and it will build automatically non optimized artifacts out of the box.
+
+## Integration Tests
+
+To keep it simple, and due technical differences between what we had before and what we actually need for core (special headers, multiple interpreters, different bootstrap logic), core integration tests can be performed simply running:
```

Comment:
As it is I feel like this text is not that useful to the "normal" reader since it's comparing PyScript and core (after). 

I'd try to give the amount of info needed for a dev to understand the rationale